test/e2e/storage/persistent_volumes-local.go

```diff
@@ -307,6 +311,37 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("when using SELinux labels", func() {
+		It("a second pod using a different label should not be able to read or write to the same volume", func() {
```

Comment:
Sorry for the confusion.  I think selinux and fsgroup mechanisms were not designed with sharing volumes in mind, so there are many issues which your work has been uncovering.  Let's have a meeting with @rootfs, @liggitt and any others to figure out what should be our way forward.